client-app/shared/wishlists/components/wishlist-line-items.vue

```diff
@@ -7,32 +7,55 @@
     removable
     @remove:items="$emit('remove:items', $event)"
   >
-    <template #default="{ item }">
-      <VcAddToCart
-        v-if="!item.deleted"
-        class="w-full"
-        :model-value="item.quantity"
-        :min-quantity="item.minQuantity"
-        :max-quantity="item.maxQuantity"
-        :available-quantity="item.inStockQuantity"
-        :count-in-cart="item.countInCart"
-        :disabled="addToCartDisabled(item)"
-        :is-in-stock="item.availabilityData?.isInStock"
-        @update:model-value="changeItemQuantity(item, $event)"
-        @update:cart-item-quantity="changeCartItemQuantity(item, $event)"
-        @update:validation="setValidationStatus(item, $event)"
-      />
-
-      <div class="mt-1.5 flex gap-1.5">
-        <InStock
-          :is-in-stock="item.availabilityData?.isInStock"
-          :is-available="!item.deleted"
-          :quantity="item.availabilityData?.availableQuantity"
-          :is-digital="item.productType === ProductType.Digital"
-        />
-
-        <CountInCart :product-id="item.productId" />
-      </div>
+    <template #titles>
```

Comment:
needs to make this space in titles 
![image](https://github.com/VirtoCommerce/vc-theme-b2b-vue/assets/18286430/2fc7e0df-030b-4041-89c2-95c45ff5b44c)